docs/platform/howto/project-support-center.rst

```diff
@@ -0,0 +1,35 @@
+Get Support in the Aiven Console
```

Comment:
```suggestion
Get support in the Aiven console
```

Reply:
Thanks for changing the support so this is sentence case. :) I had thought Aiven Console was the proper name (i.e., should be in title case)?

Reply:
Correct :) Only the word Aiven should be in uppercase and the initial sentence.

Reply:
Sorry, I meant Console should also be uppercase according to our internal Official Aiven product name guidelines. (Will share in DM.)

Reply:
Good point. If you are using in a text, you can use as follows: `Aiven Console <https://console.aiven.io>`_ so it goes with our guidelines. You can open an Issue to discuss whether we should replace in the titles as well to follow the guidelines. Maybe @TibsAtWork can help us out here.